draft-bbs-signatures.md

```diff
@@ -1309,6 +1309,8 @@ The suite of `BLS12381G1_XOF:SHAKE-256_SSWU_R0_` is defined as follows:
 
 * p: 0x1a0111ea397fe69a4b1ba7b6434bacd764774b84f38512bf6730d2a0f6b0f6241eabfffeb153ffffb9feffffffffaaab
 
+* r: 0x73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff00000001
```

Comment:
Should we move this to the (bbs) ciphersuite maybe?? The [H2C suites](https://www.ietf.org/archive/id/draft-irtf-cfrg-hash-to-curve-16.html#name-defining-a-new-hash-to-curv) don't have an `r` parameter.